Okta.Xamarin/Okta.Xamarin/OktaContext.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Okta.Xamarin
+{
+	public class OktaContext<Cl, Co> : OktaContext 
```

Comment:
Is it possible to use more meaningful names here? Client and Config instead of Cl and Co?